### DISCUSSION TITLE
URL link is always clickable

### DISCUSSION THREAD

**Discussion by unokim-dev** [community/community#49586] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

If I wrap links with backticks in issue, it is still clickable, which is not in here. Watch this:

`https://this.is/just/an/example`

```
http://i.dont/wanna/click/this
```

It seems no problem here, these aren't clickable as intended, but these are in issue!

<img width="762" alt="image" src="https://user-images.githubusercontent.com/111929571/223934315-f7f7bc70-0afd-4088-ad39-ecbffd7197f9.png">

Looking at the markup, two texts are wrapped with `<pre>`, `<code>` tags and transcoded to anchor(`<a>`) tag.
Well we aren't supposed to render the things including markups inside `<pre>` or `<code>` tag, are we?
So this is definitely weird action. I'm not using backtick markdown because it looks pretty, but because it is a code.

**Comment by Anmol-Baranwal** [comment]
> For me, it does work in issues.

Share the link, so I can help properly.

**Comment by ghost** [comment]
Hi there! It looks like you're experiencing an issue with links in your issue comments. From what you've described, it seems like links are still clickable even when you wrap them with backticks. This can be frustrating when you're trying to share a link as code.

One thing to note is that backticks are commonly used for inline code, and not necessarily for formatting links. However, I understand that you're using them to convey that the link is meant to be code and not a clickable hyperlink.

It's true that text within `<pre>` and `<code>` tags should not be rendered with HTML formatting, so it's odd that your links are still appearing as clickable hyperlinks. One potential workaround could be to use URL encoding for your links. For example, `https://this.is/just/an/example` would become `https%3A%2F%2Fthis.is%2Fjust%2Fan%2Fexample`. This would prevent the link from being rendered as a hyperlink while still allowing others to copy and paste the link.

Alternatively, you could try using a different Markdown syntax to display your links as code. For example, you could use the backslash escape character (\) before each special character in the link. Here's an example: `\https://this.is/just/an/example\`

I hope this helps! If you're still experiencing issues, feel free to reach out to the GitHub support team for further assistance.
